packages/vda-common/src/contract.ts

```diff
@@ -1,4 +1,86 @@
-// import * as fs from 'fs';
+import { BlockchainAnchor, Network } from "@verida/types";
+import { mapDidNetworkToBlockchainAnchor } from "./utils";
+import { NetworkDefinition } from "@verida/types";
+
+export interface CONTRACT_INFO {
+    abi: any,
+    address: string
+}
+
+const MYRTLE = Network.MYRTLE
+const BANKSIA = Network.BANKSIA
+
+const MAINNET = BlockchainAnchor.MAINNET.toString()
+const TESTNET = BlockchainAnchor.TESTNET.toString()
+const DEVNET = BlockchainAnchor.DEVNET.toString()
+
+export const NETWORK_DEFINITIONS: Record<Network, NetworkDefinition> = {
+    [MYRTLE]: {
+        id: Network.MYRTLE,
+        label: "Myrtle",
+        isMainnet: true,
+        anchoredBlockchain: BlockchainAnchor.POLPOS,
+        tokenAddress: "0x683565196C3EAb450003C964D4bad1fd3068D4cC",
+        storageNodeRegistryAddress: null,
+        nameRegistryAddress: "0xc9ce048b464034C53207Bf120bF85f317fdb38C8",
+        didLinkageAddress: "0x5916F97e31B77884d81bdA875b7686A988E0d517"
+    },
+    [BANKSIA]: {
+        id: Network.BANKSIA,
+        label: "Banksia",
+        isMainnet: false,
+        anchoredBlockchain: BlockchainAnchor.POLAMOY,
+        tokenAddress: "0x745Db51351015d61573db37bC16C49B8506B93c8",
+        storageNodeRegistryAddress: "0x044B2D754923e529A780eFcc085B03ee022e3364",
+        nameRegistryAddress: "0x1dD6AAc1858100091BEbb867C7628DA639F7C16E",
+        didLinkageAddress: "0xF394a23dc6777cAB3067566F27Ec5bdDD2D0bD2A"
+    },
+    [Network.DEVNET]: {
+        id: Network.DEVNET,
+        label: "Banksia",
+        isMainnet: false,
+        anchoredBlockchain: BlockchainAnchor.POLAMOY,
+        tokenAddress: "0x745Db51351015d61573db37bC16C49B8506B93c8",
+        storageNodeRegistryAddress: "0x044B2D754923e529A780eFcc085B03ee022e3364",
+        nameRegistryAddress: "0x1dD6AAc1858100091BEbb867C7628DA639F7C16E",
+        didLinkageAddress: "0xF394a23dc6777cAB3067566F27Ec5bdDD2D0bD2A"
+    },
+    [Network.LOCAL]: {
+        id: Network.LOCAL,
+        label: "Local",
+        isMainnet: false,
+        anchoredBlockchain: BlockchainAnchor.POLAMOY,
+        tokenAddress: "0x745Db51351015d61573db37bC16C49B8506B93c8",
+        storageNodeRegistryAddress: "0x044B2D754923e529A780eFcc085B03ee022e3364",
+        nameRegistryAddress: "0x1dD6AAc1858100091BEbb867C7628DA639F7C16E",
+        didLinkageAddress: "0xF394a23dc6777cAB3067566F27Ec5bdDD2D0bD2A"
+    },
+}
+
+export const BLOCKCHAIN_CHAINIDS: Record<BlockchainAnchor, string> = {
```

Comment:
The blockchain hex ids are defined in two places, risk of unsync and confusion on why one to use.
See other comment regarding a blockchain definition object.